src/main/java/omero/gateway/util/OriginalMetadataParser.java

```diff
@@ -0,0 +1,176 @@
+/*
+ *------------------------------------------------------------------------------
+ *  Copyright (C) 2013-2023 University of Dundee & Open Microscopy Environment.
```

Comment:
New class so 2013 is not needed

Reply:
I'll remove it. It was pretty much a copy/paste of your code from 2013, just into a new separate class, hence the date and author.